### PR TITLE
Generate primary and secondary records into CSVs

### PR DESCRIPTION
**Pull request by victorx716**

### CODE REVIEW

**Review by jasonto510 — COMMENTED** [review]
Everything looks fine, I've heard that faker takes a long time to seed your database though. It might be a place to optimize later.

File: database/generator.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+let string = "";
+ws.write('amenity_id, name, appeal, category, common, description, img_url \n');
+for(let i=1; i<= 35000000; i++) { 
+    let name = faker.commerce.product();
+    let appeal = Math.floor(Math.random() * 10);
+    let category = faker.commerce.department();
+    let common = Math.round(Math.random());
+    let description = faker.lorem.sentence();
+    let img_url = faker.image.image();
+    string += `${i}, ${name}, ${appeal}, ${category}, ${common}, ${description}, ${img_url} \n` ;
```
Comment (paul-do) [inline database/generator.js]: spacing in strings MAY affect what you receive when you parse

File: database/homes.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const wstream = fs.createWriteStream('homes.csv');
+
+let string = "";
+wstream.write('id, created_at \n');
+for (let i=0; i<= 10000000; i++) {
+  let rentedBefore = Boolean(Math.round(Math.random()));
+  string += `${i}, ${rentedBefore} \n` ;
```
Comment (paul-do) [inline database/homes.js]: Spacing here as well

**Comment by paul-do** [comment]
Everything looks good but I would double-check the spacing on the parse back from the csv file before moving forward

**Pull request merged**